src/app/login/login.component.html

```diff
@@ -0,0 +1,24 @@
+<div id='description-container'>
+	<div id='description'>
+		Team Mkr is a platform to connect with other developers.
+		Find others who want to work on similar skills or projects
+		and start projects with them.
+	</div>
+</div>
+<form id='login-box'>
```

Comment:
This should also be a separate component from the page so we can use it anywhere we'd like